### PR TITLE
Fix handling of unicode characters

### PR DESCRIPTION
**Pull request by lukelee1124**
Fixes #20061

### CODE REVIEW

File: pages/import_issues_page_col_set.php [hunk 1]
```diff
@@ -125,7 +125,7 @@
 			echo sprintf( plugin_lang_get( 'column_number' ), $i + 1);
 		}
 		else {
-			echo prepare_output($t_column_title[$i]);
+			echo $t_column_title[$i];
```
Comment (vboctor) [inline pages/import_issues_page_col_set.php:128]: What is the reason from removing the prepare_output() which likely does the escaping?

Comment (lukelee1124) [inline pages/import_issues_page_col_set.php:128]: @vboctor this output is already convert as UTF-8,if use prepare_output() again will make the previous processing in vain

File: Csv_import.php [hunk 2]
```diff
@@ -5,7 +5,7 @@ function register() {
 		$this->name = plugin_lang_get( 'title' );
 		$this->description = plugin_lang_get( 'description' );
 		$this->version = '1.4.0';
-		$this->requires = array( 'MantisCore' => '1.2.0' );
+		$this->requires = array( 'MantisCore' => '1.3.0' );
```
Comment (atrol) [inline Csv_import.php:8]: Is there a reason that the plugin does no longer support 1.2.x?

Comment (lukelee1124) [inline Csv_import.php:8]: @atrol my local is 1.3.X,here is a mistake,i forget to come back.

**Pull request closed without merging**